MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -234,21 +241,50 @@ object VisionManager : BaseVisionManager {
      * @param visionEventsListener: listener for [VisionManager]. Is held as a strong reference until [stop] is called.
      */
     @JvmStatic
+    @Deprecated(
+        "Will be removed in 0.9.0. Use start() and var visionEventsListener: VisionEventsListener? instead",
+        ReplaceWith("VisionManager.start()")
+    )
     fun start(visionEventsListener: VisionEventsListener) {
-        delegate.checkManagerCreated()
         if (delegate.isStarted) {
```

Comment:
Extract common code for `start` methods?

Reply:
because of different flow(need strong listener link for old one) let's keep as it is